generators/generator-bot-adaptive/generators/app/index.js

```diff
@@ -192,7 +187,10 @@ module.exports = class extends BaseGenerator {
     appSettings.runtime.key = `adaptive-runtime-${platform}-${integration}`;
     appSettings.runtime.command = {
       [platforms.dotnet]: `dotnet run --project ${botName}.csproj`,
-      [platforms.js]: 'node index.js',
+      [platforms.js]: {
+        [integrations.functions]: 'func start',
```

Comment:
I thought @benbrown had indicated we were having issues for using dotnet run on Functions for .NET as well. Do we not need to update the command for .NET Functions also, or did I misunderstand?

Reply:
I didn't test that change, it's likely we will but I'm not sure 100% and don't have the environment set up to run .NET functions apps. I can look into that today though if it's an issue.

Reply:
@peterinnesmsft I've been getting some errors trying to run `func start` with a dotnet project locally, I think it would be best for someone more versed in .NET to try to resolve.

Reply:
I'll take this on this week. Let's keep this train moving and get this change in - I'll follow up with another change to support .NET Functions once I've had time to sit down with it and test it out.